client/spa/js/instructor/instructor.html

```diff
@@ -0,0 +1,21 @@
+<style type="text/css">
+.main {
+  margin: auto;
+  max-width: 980px;
+}
+#result {
+  padding: 1em 0;
+}
+.success {
+  color: green;
+}
+</style>
+<section id="instructor">
+  <div class="main container">
+    <h1><%- firstName %> <%- lastName %></h1>
+    <p><%- skills %></p>
+    <button class="edit">Edit</button>
```

Comment:
Ok, I'll add a different class for my templates so it doesn't overwrite yours and lessen the # of lines.